capistrano-s3.gemspec

```diff
@@ -18,6 +18,9 @@ Gem::Specification.new do |s|
   s.cert_chain  = ['certs/aleksandrs-ledovskis.pem']
   s.signing_key = File.expand_path("~/.ssh/gem-private_key.pem") if $0 =~ /gem\z/
 
+  # Min rubies
+  s.required_ruby_version = ">= 2.0"
```

Comment:
Pro/Cons of pushing the bar all the way to 2.1?

+ 2.0 has [been EOL'ed](https://www.ruby-lang.org/en/news/2016/02/24/support-plan-of-ruby-2-0-0-and-2-1/) for a good 6 months
+ Having 2.0 as a lowest acceptable version necessitates its inclusion in CI matrix